build-system/compile/typescript.js

```diff
@@ -31,7 +28,9 @@ const {red} = require('kleur/colors');
  * @param {string} srcFilename
  * @return {!Promise}
  */
-exports.transpileTs = async function (srcDir, srcFilename) {
+async function transpileTs(srcDir, srcFilename) {
```

Comment:
Two nonblocking questions:
 - what do we use `transpileTs` for? I already transferred the "New Server" to use esbuild.
- Is there an "Old Server" we can delete? If not can we rename the new server --> server

Reply:
It's an as yet unused function called `compileTs` that was added a long time ago in the hopes of gradually rewriting AMP in TS. I'd be fine with removing it, really. 

https://github.com/ampproject/amphtml/blob/11b87675600c73016a0016cdd980155508c7352e/build-system/tasks/helpers.js

Reply:
And everything in `app.js` is our current (old) server. We had plans for rewriting it as a shared team effort, but haven't been able to prioritize it given the sheer size. 

https://github.com/ampproject/amphtml/issues/24333